static/js/app.js

```diff
@@ -256,7 +256,7 @@ $(function () {
 
             _.each(this.oms.markersNearAnyOtherMarker(), function (marker) {
                 // hide markers using markers.css
-                marker.title = '';
+                marker.view.model.get("currentlySpiderfied")? marker.setTitle(marker.view.getTitle("single")) : marker.setTitle(marker.view.getTitle("multiple"));
```

Comment:
There is no advantage to the conditional statement here, and the line is too long.
I think it's only beneficial when it helps avoid repetition.
So you could actually do

    marker.setTitle(marker.view.getTitle(marker.view.model.get("currentlySpiderfied") ? "single" : "multiple"));

Or just use an `if`.